Banstein/africar-frontend
# Setup project

Hello Reviewer ✋🏽 
## Description of changes made ♻️

> In this PR  I have added the react project setup using the `CREATE-REACT-APP` command
I have set up the project for design and styling using tailwindcss
All linters, and stylelint are also setup

#### Type of change

> **Please select the relevant options**


#### How Has This Been Tested?

> **Please describe the tests that you ran to verify your changes**


#### How should this be manually tested?

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo]()
- Check out this branch by `branch here`
- Run `yarn or npm install to install the project dependencies
- Run `npm run start:africar` to start the development server
- Run `npm start` for production serve to build the project

#### Checklist


#### Screenshots

|        BEFORE        |       AFTER       |
| :------------------: | :---------------: |
| 1. [Given Figma design](https://www.figma.com/file/jP4w1VZFvauBSiByZTrY4c/AFR-SG-MAP-V2?node-id=314%3A5456) | 2. Implemented: `Run ====> npm run start:africar` |
|
